public/js/track.js

```diff
@@ -53,6 +53,10 @@ $('#playlistModal').on('hide.bs.modal', function () {
     $('#collapsePlaylistMessage').collapse('hide');
 })
 
+$(document).on('click', '.alert-close', function() {
```

Comment:
What does this do?

Reply:
Hides the alert!

Reply:
Oh so anywhere they click on the screen the alert goes away? Do we want the user to go up to the alert and click OK to dismiss the alert? Cause what if they accidentally click on screen and they missed the alert message

Reply:
Nope I had to create my own "close" button for the alert because the bootstrap one was quite buggy due to the dynamic rendering! The alert only goes away when they click "X" on the top right, have you been able to check into my branch to see?

Reply:
ohh okok I see sorry my bad

How come we can't do like this? It works for me
![image](https://user-images.githubusercontent.com/49849754/94350297-ca10d480-0001-11eb-844a-1663ff36c576.png)

Reply:
Your snippet only worked for me when I wasn't rendering it dynamically- otherwise it was really buggy!